src/scintillaeditor.cpp

```diff
@@ -996,3 +996,14 @@ void ScintillaEditor::prevBookmark()
 	if (line != -1)
 		qsci->setCursorPosition(line, index);
 }
+
+void ScintillaEditor::jumpToNextError()
```

Comment:
With now 3 mostly identical functions, it would be nice to have just one implementation. Maybe something along the lines of https://godbolt.org/z/wpniJJ ?

Reply:
Merging the other pull requests did cause a conflict again. Fixing via #3063...